README.md

```diff
@@ -22,7 +21,7 @@ This library requires that the `nvidia_drm` kernel module is [configured with th
 
 # Codec Support
 
-Only decoding video is currently supported.
+Hardware encoding only, decoding is [not supported](/../../issues/116).
```

Comment:
You've switched the meaning here. Only decoding is supported, not encoding.

Reply:
Good catch 🤦🏻